docs/lang/articles/reference.md

```diff
@@ -176,22 +176,72 @@ compile time.
 
 #### Calls
 
+```
```

Comment:
1. Let's use single space and "" for consistence.
2. We support positional arguments as shown in https://docs.python.org/3/reference/expressions.html#calls, so the rules need to be updated.
3. Our call implementation (https://github.com/taichi-dev/taichi/blob/0e715318cb0b6a19fd2313e930114d63e4960283/python/taichi/lang/ast/ast_transformer.py#L409) includes many cases, so a more detailed description is need:
    - Calling a Python function
    - Calling a Taichi function
    - Calling a Taichi builtin function
    - Calling a type annotation for literals